definitions/npm/react-jss_v6.1.x/flow_v0.30.x-/react-jss_v6.1.x.js

```diff
@@ -8,6 +8,8 @@ declare module 'react-jss' {
 
   declare type Null = null | void;
 
+  declare type Dictionary = { [key: string]: {} }
```

Comment:
This is rather done for the documentation purposes. Object type is too wide and includes functions and arrays (not sure about the latter though). So I just wanted to underline that we expect a dictionary there